### ISSUE TITLE
Link Verification and Type Inference tickets in the spec

### ISSUE THREAD

**Issue by burmako** [openxla/stablehlo#612]
So far, whenever we encountered misalignments between the spec and verifiers / type inference implemented in the StableHLO dialect, we opened tickets and marked Verification and Type Inference columns in status.md as "revisit".

This is useful, but in practice this loses some important information. When seeing those revisits in status.md, I'm having a hard time remembering what was the reason for them. Similarly, when reading the spec, I sometimes struggle to remember the status of the StableHLO dialect with respect to it.

As discussed earlier, we should:
  1. Go through all Verification and Type Inference tickets.
  2. Identify the tickets which describe misalignments with the spec (not new features, but bugs!).
  3. Mention these tickets in specs, right next to relevant parts of Inputs/Outputs/Constraints.

**Comment by burmako** [comment]
I've briefly looked through all these tickets, and while most of them affect individual ops or small groups of ops, there are tickets that affect lots of ops: 
  * #22.
  * #369.
  * #481.

I think that for the three tickets mentioned above (and potentially other tickets that we might discover in the coming days) we shouldn't call them out in the spec to avoid spamming and similarly we shouldn't flip Verification / Type Inference to "revisit" because of them.

**Comment by burmako** [comment]
Fixed in https://github.com/openxla/stablehlo/pull/1076 which links almost all Frontend Contract tickets (except for four tickets for which I haven't come up with an ergonomic way to call them out), as well as select tickets from other projects. 

When originally opening this ticket, I was mainly thinking about linking Specification Compliance tickets, but then I realized that linking them in the spec is not a great idea. Frontend Contract tickets are interesting because they identify gaps in the spec - something that's likely very relevant to readers of the spec - whereas Specification Compliance tickets talk about gaps in the StableHLO dialect, which is just one possible implementation.
